src/plugin/plugin_pdo.rs

```diff
@@ -198,8 +198,17 @@ fn after_hook(
             );
         }
     } else if let Some(obj) = return_value.as_mut_z_obj() {
-        if obj.get_class().get_name() == &"PDOStatement" {
-            return after_hook_when_pdo_statement(get_this_mut(execute_data)?, obj);
+        let cls=obj.get_class().get_name().to_str()?;
+        match cls {
+            "PDOStatement" => {
+                return after_hook_when_pdo_statement(get_this_mut(execute_data)?, obj);
+            }
+            r"Doctrine\DBAL\Driver\PDO\Statement" => {
+                return after_hook_when_pdo_statement(get_this_mut(execute_data)?, obj);
+            }
+            _ => {
+                debug!(cls, "not a valid class");
+            }
```

Comment:
Is `Doctrine\DBAL\Driver\PDO\Statement` the subclass of `PDOStatement`?
Maybe it's better to [is_instance_of](https://docs.rs/phper/latest/phper/classes/struct.ClassEntry.html#method.is_instance_of) to detect.

Reply:
Indeed this is a much better approach and it works for my case. Addressed in next commit, run cargo fmt and went back from match to if/else

Reply:
Good job!